models/safe/zkevm/safe_zkevm_matic_transfers.sql

```diff
@@ -0,0 +1,79 @@
+{{
+    config(
+        materialized='incremental',
+
+        alias = 'matic_transfers',
+        partition_by = ['block_month'],
+        unique_key = ['block_date', 'address', 'tx_hash', 'trace_address'],
+        on_schema_change='fail',
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook = '{{ expose_spells(
+                        blockchains = \'["zkevm"]\',
+                        spell_type = "project",
+                        spell_name = "safe_matic_transfers",
+                        contributors = \'["danielpartida"]\') }}'
```

Comment:
By the way, @jeff-dude  do we need to make the new post_hook format consistent for all new spells?

Reply:
we should strive to add the arg names in front like this moving forward, yes

Reply:
however, for `spell_name`, it should actually be project name (i.e. `safe`) to keep the data explorer clean.
the naming `spell_name` is a bit confusing, we should likely update in future cleanup.
similarly, if `spell_type = sector` rather than project, we'd want sector name in `spell_name`